survey/src/main/kotlin/com/surveyor/survey/controllers/SurveyController.kt

```diff
@@ -0,0 +1,59 @@
+package com.surveyor.survey.controllers
+
+import com.surveyor.survey.services.QuestionService
+import com.surveyor.survey.services.SurveyService
+import java.util.*
+import org.springframework.http.ResponseEntity
+import org.springframework.http.HttpStatus
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.web.bind.annotation.*
+import org.springframework.validation.BindingResult
+import javax.validation.Valid 
+import com.surveyor.survey.models.SurveyRequest
+import com.surveyor.survey.models.SurveyDTO
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.surveyor.survey.entities.Answer
+import com.surveyor.survey.mappers.SurveyMapper
+
+
+@RestController
+@RequestMapping("/survey")
```

Comment:
```suggestion
@RequestMapping("/surveys")
```

Plural nouns make the API more intuitive and consistent because they imply a collection of resources.